src/components/Accounts.tsx

```diff
@@ -53,8 +53,8 @@ const Accounts = ({ className }: Props) => {
   const onChange = (value: string, chain: string) => {
     setCurrentAccount(value, chain);
   };
-
   const renderAccounts = (chain: string) => {
+    console.log('chain is:', chain);
```

Comment:
stray log

Reply:
Its not stray - Its called Jack and its my console log - I pet it and feed it 😄